docs/for-devs/console/workloads.md

```diff
@@ -28,111 +18,45 @@ All known Workloads of the team are listed here.
 
 | Property      | Description                                            |
 | ------------- | ------------------------------------------------------ |
-| Workload Name | The name of the workload                               |
-| Type | The type of the workload (deployment/ksvc/custom)               |
-| Argocd | Link to the ArgoCD application in the ArgoCD UI               |
+| Name | The name of the workload                               |
+| Argocd | Link to the Argo CD application in the Argo CD UI               |
 
 
 ## Create a Workload
 
-1. In the right menu click on `Workloads` and then on `Create workload`.
+Before creating a workload from the developer catalog, we'll need the `repository` and `tag` of the image you would like to use.
 
-2. Select the workload type
+1. Go to the list of Builds and add the `repository` of the build to use to your clipboard. Remember the tag.
 
-- Regular application: will use the [Otomi deployment Helm chart](https://github.com/redkubes/otomi-charts)
-- Function as a Service: will use the [Otomi Knative service Helm chart](https://github.com/redkubes/otomi-charts)
-- Bring your own Helm chart: use your own (custom) Helm chart
+2. Go to `Workloads` in the right menu and click on `New Workload`
 
-### Regular application
+3. Add a `Name` for the workload
 
-3. Enter a name for the workload
-4. Provide [Basic](#basic-values) or [Advanced](#advanced-values) values configuration
-5. Click `Next`
-6. Review the Values used to install the chart. Optionally add more values. See [here](https://github.com/redkubes/otomi-charts/blob/main/deployment/values.yaml) for all supported values
-7. Click `Submit`
+4. Select a template from the catalog
 
-Now click on `application` in the `Argocd` column of the workload in the list of workloads. Note that an ArgoCD application is created to deploy the workload.
+5. (Optional) Configure the `Auto image updater`. Default is `Disabled`.
 
-### Function as a Service
+The Auto Image Updater will (based on the update strategy) automatically update the container images of a workload.
 
-1. Enter a name for the workload
-2. Provide [Basic](#basic-values) or [Advanced](#advanced-values) values configuration
-3. Click `Next`
-4. Review the Values used to install the chart. Optionally add more values. See [here](https://github.com/redkubes/otomi-charts/blob/main/ksvc/values.yaml) for all supported values
-5. Click `Submit`
+Select the update strategy. Choose between:
 
-Now click on `application` in the `Argocd` column of the workload in the list of workloads. Note that an ArgoCD application is created to deploy the workload.
-### BYO Helm chart
+- Digest: Update to the most recent pushed version of a given tag. Requires to provide a `repository` and a `tag`.
```

Comment:
From the description I would understand that it is actually latest.

Reply:
Digest: Inspects a single tag in the registry for changes, and updates the image on any change to the previous state